app/lib/log_request_params.rb

```diff
@@ -11,7 +12,12 @@ def self.included(base)
   end
 
   def add_params_to_request_store
-    keys_to_ignore = IGNORE_PARAMS.map { |regexp| params.keys.grep(regexp) }.flatten.uniq
-    RequestLocals.store[:params] = params.except(*keys_to_ignore)
+    if request.get?
+      keys_to_ignore = IGNORE_PARAMS.map { |regexp| params.keys.grep(regexp) }.flatten.uniq
+      RequestLocals.store[:params] = params.except(*keys_to_ignore)
```

Comment:
Use `query_params` instead of `params`, so we don't include POST params (and we don't need the conditional).

Reply:
Adding path_parameters, too, which gives us :controller and :action in the params as before, but also params set from the url/path (e.g. course_id from /support/courses/:course_id)